app/src/main/res/layout/fragment_device_detail.xml

```diff
@@ -15,7 +15,7 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:orientation="vertical"
-            android:paddingBottom="@dimen/detail_vertical_margin"
+            android:paddingBottom="70dp"
```

Comment:
Could you modify src/main/res/values/dimens.xml instead?
And also make the title of this commit clearer (for instance, what component do you add padding to)

Reply:
I will do that tomorrow. I wasn't sure it isn't used by something else too. Should I implemented [the 3 dot menu](https://github.com/AsteroidOS/AsteroidOSSync/issues/75#issuecomment-490250689) as well?

Reply:
Yes, please do